src/com/sittinglittleduck/DirBuster/Worker.java

```diff
@@ -405,7 +219,7 @@ else if(work.getBaseCaseObj().isUseRegexInstead())
                                 System.out.println("DEBUG Worker[" + threadId + "]: Found Dir (regex) " + url.toString());
                             }
                             //we found a dir
```

Comment:
With minor adjustments this (i.e. if/else statements) can be replaced with a call to ```sendResponseToManager```.